typos.toml

```diff
@@ -23,3 +25,5 @@ default.extend-ignore-re = ["\"eyJ[a-zA-Z0-9_\\-\\.]+\""]
 Ons = "Ons"
 Lsat = "Lsat"
 quizs = "quizs"
+Aggrement = "Aggrement"
```

Comment:
I don't think we should allow this typo across the whole code base when it is only present in 1 file. Can we make a narrower exclude?